inc/taxonomies.php

```diff
@@ -1503,6 +1503,20 @@ function cptui_update_taxonomy( $data = [] ) {
 		return 'error';
 	}
 
+	$must_have_keys = ['object_types', 'rewrite', 'rewrite_slug', 'name',
```

Comment:
Can we turn this into a function similar to `cptui_reserved_taxonomies()` where it returns this array? No need to add an `apply_filters()` for it, just have it return the array.

Could either still assign to `$must_have_keys` or it could replace the variable and just be used as such:

```
foreach( my_function_name() as $key ) {
...
}
```

The reason for this request is to make the array more usable in the future, if need arises.